desktop/src/main/kotlin/com/github/picture2pc/desktop/ui/main/MainScreen.kt

```diff
@@ -66,45 +57,10 @@ fun MainScreen(
                     ) { Picture() }
```

Comment:
For the picture, there are 4 Boxes inside of eachother when it only needs to be one. 3 in this file and one in Picture